common/src/systems/collision.rs

```diff
@@ -125,6 +138,7 @@ impl<'s> System<'s> for ProjectileCollisionHandler {
                     damage_event.single_write(DamageEvent { damage: *damage, target: collision.target })
             }}
 
+            log::info!("Delete entity. Collision");
```

Comment:
Ithink these messages should be on debug level. Some additional info might be useful (I don't know if enity_ids are meaningful enough.)

Reply:
Oh, it's deleted. OK